foreman.te

```diff
@@ -463,6 +480,15 @@ optional_policy(`
     ')
 ')
 
+######################################
+#
+# Apache httpd server
+#
+
+# Apache httpd reads puppet certs
+read_files_pattern(httpd_t, puppet_etc_t, puppet_etc_t)
+read_lnk_files_pattern(httpd_t, puppet_etc_t, puppet_etc_t)
+
```

Comment:
And this is the fix of the reported issue - when installing Foreman core (no Katello), httpd is configured with puppet certs. This was removed because I thought this is puppet master rule.